docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,119 @@
-# AutoGluon Cloud
-Coming Soon...
+# Deploying AutoGluon Models with AutoGluon Cloud on AWS SageMaker
+:label:`autogluon-cloud`
+
+To help with AutoGluon models training, AWS developed a set of training and inference [deep learning containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#autogluon-training-containers). 
+The containers can be used to train models with CPU and GPU instances and deployed as a SageMaker endpoint or used as a batch transform job.
+
+We provide a full end-to-end example in [amazon-sagemaker-examples](https://github.com/aws/amazon-sagemaker-examples/tree/master/advanced_functionality/autogluon-tabular-containers) repository.
```

Comment:
Why are we headlining with mentioning the old tutorial that we want to make unnecessary with this repo?

Reply:
Yea. The more I think of it, it's probably unnecessary